pyensign/ensign.py

```diff
@@ -107,8 +122,13 @@ def __init__(
         self.client = Client(connection, topic_cache=self.topics)
 
     async def publish(
-        self, topic, *events, on_ack=None, on_nack=None, ensure_exists=False
-    ):
+        self,
+        topic: str,
+        *events: Any,  # This is inferred to be any type, but might be narrowed down later.
```

Comment:
Can we remove the comment? I'd like to keep comments in the docstring so the function parameters are concise and readable. The `Any` type hint is a common thing that people can look up so I don't think we are obligated to explain that here.

Reply:
Actually if possible this should be the custom `Event` type here instead of `Any`.